reconcile/utils/aws_api.py

```diff
@@ -1091,6 +1092,23 @@ def get_tgws_details(
                 "region": region_name,
             }
 
+            if route53_associations:
+                route53 = self._account_route53_client(account["name"], region_name)
+                paginator = route53.get_paginator('list_hosted_zones')
+                zones = []
+                for page in paginator.paginate():
+                    for zone in page["HostedZones"]:
+                        if zone["Config"]["PrivateZone"]:  # TODO: add filter on tags?
```

Comment:
We'll need an agreement on this tag filtering with @dustman9000

Reply:
no need for tag filtering. AVO will be able to delete the zone even it is associated to our VPCs and there should be no other hosted zones in the service transit account